src/test/java/bio/terra/integration/DataRepoClient.java

```diff
@@ -0,0 +1,91 @@
+package bio.terra.integration;
+
+import bio.terra.model.DeleteResponseModel;
+import bio.terra.model.ErrorModel;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+/**
+ * This class holds a Spring RestTemplate
+ */
+@Component
+public class DataRepoClient {
+    @Value("${integrationtest.port}")
+    private String testPort;
+
+    @Value("${integrationtest.server}")
```

Comment:
I've been reading that it is preferable to use `@ConfigurationProperties(prefix = "integrationtest")` on the class instead of `@Value(...)` on the properties. It doesn't seem to be a big difference either way, but we should standardize on our approach to how we inject externalized configuration.

Reply:
It is generally better practice to avoid @Value because it smears the configuration all over the code, making it hard to manage. If I understand correctly, I would need a separate class to map to the configuration properties and use accessors in this code. Is that your understanding?